src/app/contribute/guidelines/components/index.html

```diff
@@ -0,0 +1,240 @@
+    </p>
+  </ol>
+
+  <stache-page-anchor>
+    Write documentation
+  </stache-page-anchor>
+
+  <p>
+    We recommend that you create reference documentation for the <a stacheRouterLink="/components">Components section</a> of the SKY UX website. Component documentation should be extensive enough to explain the features of your components, including all properties and events. In the <a href="https://github.com/blackbaud/skyux2-docs">SKY UX docs repo</a>, you create subfolders for your components in the <stache-code>src/app/components</stache-code> folder and document them in index.html files. To create action buttons for the Components section landing page, you add entries in the <stache-code>demo-components.service.ts</stache-code> file.
+  </p> 
+  
+  <p>
+    If you do not document your components, the SKY UX team will create the documentation, but we request that you <a href="https://github.com/blackbaud/skyux2-docs/issues">file an issue</a>. However, we strongly encourage you to create your own documentation because it can be useful to uncover issues with your components.
+  </p>
+
+  <stache-page-anchor>
+    Write unit tests
+  </stache-page-anchor>
+
+  <p>
+    Your unit test files should end in <stache-code>.spec.ts</stache-code> and reside in the same folders as the components that they test. For example, the test file for <stache-code>foo/foo.component.ts</stache-code> should be <stache-code>foo/foo.component.spec.ts</stache-code>. All test fixtures such as sample components, mock services, etc. should be in a fixtures folder within the component's folder. The existing codebase contains many examples of this pattern, so just follow the patterns in the existing code when you write tests.
+  </p>
+
+  <p>
+    For new components, we recommend that you reach out to the SKY UX team for a "soft" review of the initial implementation before you write unit tests. This allows you to make requested changes to your code before writing the tests.
+  </p>
+
+  <p>
+    When your tests run, code coverage results are generated and can be located in <stache-code>coverage/&lt;browser version>/index.html</stache-code>. You can launch this straight from disk and view the SKY UX unit test code coverage results in your default web browser.
+  </p>
+
+  <stache-page-anchor>
+    Write visual regression tests
+  </stache-page-anchor>
+
+  <p>
+    During continuous integration builds, SKY UX runs visual regression tests through BrowserStack using <a href="http://www.protractortest.org/">Protractor</a> and <a href="https://github.com/koola/pix-diff">pix-diff</a>. These prerequisites are automatically installed when you run <stache-code>npm run test:visual</stache-code> to create and compare screenshots in the <stache-code>skyux-spa-visual-tests/screenshots-baseline-local/</stache-code> folder.
```

Comment:
I would change this to `skyux e2e` since we can't guarantee that every repo has a definition for this NPM script.